bower.json

```diff
@@ -28,7 +28,7 @@
     "iron-fit-behavior": "PolymerElements/iron-fit-behavior#2.0-preview",
     "neon-animation": "PolymerElements/neon-animation#2.0-preview",
     "paper-styles": "PolymerElements/paper-styles#2.0-preview",
-    "web-animations-js": "web-animations/web-animations-js#^2.2.0"
+    "web-animations-js": "web-animations/web-animations-js#^2.2"
```

Comment:
This should be in `devDependencies` now, because it's not actually imported by the element itself, right?